tests/deploy_test.go

```diff
@@ -92,6 +92,21 @@ func Test_Deploy_MetaData(t *testing.T) {
 				Namespace: config.DefaultNamespace,
 			},
 		},
+		{
+			name: "Deploy with memory limit",
+			function: types.FunctionDeployment{
+				Image:       imagePath,
+				Service:     "memory-limit",
+				EnvProcess:  "env",
+				Annotations: &map[string]string{},
+				Labels:      &map[string]string{},
+				Namespace:   config.DefaultNamespace,
+				Limits: &types.FunctionResources{
+					Memory: "5M",
```

Comment:
`5Mi`?

Reply:
Changes are done and it works for both faas-netes and faasd.